colossus/src/main/scala/colossus/protocols/memcache/Memcache.scala

```diff
@@ -189,28 +204,36 @@ sealed trait MemcacheWriteCommand extends MemcacheCommand {
   def value: ByteString
   def ttl: Int
   def flags: Int
+  def casUniqueMaybe: Option[Int]
 
   def bytes(compressor: Compressor): ByteString = {
 
     val b = new ByteStringBuilder
 
     /*write commands are of the format
     We don't support the [noreply] semantics at this time
-    <COMMAND> <KEY> <FLAGS> <EXPTIME> <BYTECOUNT>\r\n<data>\r\n
+    <COMMAND> <KEY> <FLAGS> <EXPTIME> <BYTECOUNT> [<CAS UNIQUE>]\r\n<data>\r\n
 
     padding accounts for spaces, \r\n's, bytecount and exptime:
-       4 spaces
+       4 or 5 spaces, depending if <CAS UNIQUE> is present
     +  4 (2 \r\n's)
      ----
-       8 */
-
-    val padding = 8
+       8 or 9, depending if <CAS UNIQUE> is present*/
 
     val flagsStr    = ByteString(flags.toString)
     val ttlStr      = ByteString(ttl.toString)
     val dataSizeStr = ByteString(value.size.toString)
+    val casUniqueStr = ByteString(casUniqueMaybe.getOrElse("").toString)
```

Comment:
This seems to be an `Option[Int]` that has `getOrElse[String]` being called. Is that chill?

Reply:
oh hmm, i tested this in `sbt console` and it appears it is only chill because of the `#toString`:

![screen shot 2018-11-01 at 4 32 06 pm](https://user-images.githubusercontent.com/6476505/47878097-d8451e80-ddf3-11e8-97b9-ab2834f1ecd1.png)

Reply:
Yeah thats a tricky statement. casUniqueMaybe.getOrElse("") gives you any type. This is a more readable version:
``` 
val casUniqueAsStr = casUniqueMaybe match { 
  case Some(_) => casUniqueMaybe.get.toString
  case None => ""
 }
```

Reply:
interesting! personally i prefer the condensed version in the case 🐱

Reply:
The fact that we are talking about this suggests that the condensed version is not the one you should be using.

Reply:
ah okay, i did not know you felt so strongly about this! will update ^_^

Reply:
Alternative: `anIntMaybe.map(_.toString).getOrElse("")`

Reply:
Or: `antIntMaybe.fold("") { _.toString }`